internal/provider/data_source.go

```diff
@@ -50,6 +50,17 @@ your control should be treated as untrustworthy.`,
 			},
 
 			"body": {
+				Description: "The response body returned as a string. " +
+					"**NOTE**: This is deprecated, use `response_body` instead.",
+				Type:     schema.TypeString,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
```

Comment:
`schema.TypeString` does not have elements like a list, map, or set would. 👍 

```suggestion
```

Reply:
Have removed `Elem` from `url`, `body` and `response_body` as all are `TypeString`.